packages/manager/modules/log-to-customer/src/index.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerLogToCustomerLazyLoading';
+
+angular.module(moduleName, ['oc.lazyLoad', 'ui.router']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider.state('pci.projects.project.kubernetes.details.logs.**', {
+      url: '/logs',
+      lazyLoad: ($transition$) => {
+        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+
+        return import('./live-tail/module').then((mod) => {
+          return $ocLazyLoad.inject(mod.default || mod);
+        });
+      },
+    });
+  },
+);
```

Comment:
Why define a PCI state in Logs to customer module. Logs to customer module can export a component and in PCI, you can create a state and use this component.